[2021-12-01 16:49:17,875] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: FrequencyCount.map2 manual__2021-12-01T16:49:14.737488+00:00 [queued]>
[2021-12-01 16:49:17,887] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: FrequencyCount.map2 manual__2021-12-01T16:49:14.737488+00:00 [queued]>
[2021-12-01 16:49:17,887] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-01 16:49:17,887] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2021-12-01 16:49:17,887] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-01 16:49:17,899] {taskinstance.py:1262} INFO - Executing <Task(MapOperator): map2> on 2021-12-01 16:49:14.737488+00:00
[2021-12-01 16:49:17,905] {standard_task_runner.py:52} INFO - Started process 166 to run task
[2021-12-01 16:49:17,908] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FrequencyCount', 'map2', 'manual__2021-12-01T16:49:14.737488+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/mapred.py', '--cfg-path', '/tmp/tmpkn9d56qi', '--error-file', '/tmp/tmpxyvywwj0']
[2021-12-01 16:49:17,908] {standard_task_runner.py:77} INFO - Job 73: Subtask map2
[2021-12-01 16:49:17,946] {logging_mixin.py:109} INFO - Running <TaskInstance: FrequencyCount.map2 manual__2021-12-01T16:49:14.737488+00:00 [running]> on host 1da30e33d42e
[2021-12-01 16:49:17,990] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FrequencyCount
AIRFLOW_CTX_TASK_ID=map2
AIRFLOW_CTX_EXECUTION_DATE=2021-12-01T16:49:14.737488+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-01T16:49:14.737488+00:00
[2021-12-01 16:49:18,214] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=FrequencyCount, task_id=map2, execution_date=20211201T164914, start_date=20211201T164917, end_date=20211201T164918
[2021-12-01 16:49:18,240] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-01 16:49:18,271] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
